src/System.Data.SqlClient/src/Microsoft/SqlServer/Server/ExtendedClrTypeCode.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+
+//------------------------------------------------------------------------------
```

Comment:
Nit: Some of these files seem to have unnecessary lines between the license header and the start of the code.